storybook/storybook-react/src/MegaMenu/MegaMenu.stories.tsx

```diff
@@ -0,0 +1,288 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Grid, Heading, Link, MegaMenu, UnorderedList } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Navigation/Mega Menu',
+  component: MegaMenu,
+  args: {
+    children: 'Nieuw component',
+  },
+  parameters: {
+    layout: 'fullscreen',
+  },
```

Comment:
Does this do anything?

Reply:
This removes the extra 1rem padding of the story, especially useful when you are working with a grid to avoid double padding.